### PR TITLE
[IMP] [15.0] hr_holidays: split function get employees

### PR DESCRIPTION
**Pull request by locdang8**
-Currently when i create leave with holiday type department or company, it will create leave for all employee of that department and company. I am building a feature to be able to remove unwanted employees, separating this code I think will be necessary for me or other people who want customize. Please consider helping me with this problem. 

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_holidays/models/hr_leave.py [hunk 1]
```diff
@@ -1136,6 +1136,18 @@ def action_approve(self):
     def _get_leaves_on_public_holiday(self):
         return self.filtered(lambda l: l.employee_id and not l.number_of_days)
 
+    def _get_employee_with_holiday_type(self):
```
Comment (Bertrand2) [inline addons/hr_holidays/models/hr_leave.py:1139]: As we return multiple employees, let's make the method name reflect that
```suggestion
    def _get_employees_from_holiday_type(self):
```

**Review by Bertrand2 — COMMENTED** [review]
Thanks for your PR! This is indeed a very good addition.
Have a nice day!

File: addons/hr_holidays/models/hr_leave.py (hunk 1 above)
Comment (locdang8) [reply addons/hr_holidays/models/hr_leave.py]: Thank you so much for looking into it, I hope it will be merged soon !!

**Comment by sofiagvaladze** [comment]
robodoo r+

**Pull request closed without merging**